release/test-protobuf-structure.sh

```diff
@@ -55,6 +55,7 @@ com/daml/ledger/api/v1/ledger_configuration_service.proto
 com/daml/ledger/api/v1/ledger_identity_service.proto
 com/daml/ledger/api/v1/ledger_offset.proto
 com/daml/ledger/api/v1/package_service.proto
+com/daml/ledger/api/v1/page_tokens.proto
```

Comment:
This should be moved to an `Api` server internal location, as we do not want users to parse the tokens. They should consider them opaque.

@nmarton-da can you advise wrt where to place that file, and code structure overall?

Reply:
We shouldn't use the same namespace as the public ledger api no.

If we go by the content of https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/protobuf/daml/platform/index.proto, then a decent option seems to be:
```
package daml.platform.apiserver;

option java_package = "com.daml.platform.apiserver";
```

No need to define the `csharp_namespace` as this .proto file is internal to our api server.

Reply:
@nmarton-da : what do you think?

Reply:
I agree with @meiersi-da up there, best would be to follow the index.proto in this case:
* no csharp namespace needed
* having the package as apiserver
Only thing is different to index.proto, that this is a API artefact, while index.proto is a persistent one, so probably proto backwards compatibiliy checks will apply here as well. But that is only stronger than what we need here, which is ok until it is not in the way.

Reply:
Ok